### ISSUE TITLE
feat: Support bootc

### ISSUE THREAD

**Issue by gmpinder** [blue-build/cli#200]

**Comment by rriemann** [comment]
@xynydev pointed me to this issue. For the ideas described here below, bootc support would be essential:

https://eu-os.gitlab.io/goals#goals

**Comment by rriemann** [comment]
Hi @gmpinder ,

With dnf5 support, I thought we got basically bootc support. What's missing yet?

**Comment by gmpinder** [comment]
> I'm not expert enough to understand the implications. Can I already today use bluebuild to build an iso on top of the ublue/AlmaLinux bootc containers?
> 
> I understand that's what winblues/blue95 does.

Yes you can. The work I'm doing is for local builds, not CI builds
